### ISSUE TITLE
Buildtracker support for multiple applications

### ISSUE THREAD

**Issue by abhighyaa-jain** [paularmstrong/build-tracker#97]
# Problem

In my organization, we have several microservices, so if build tracker could be configured in a way that at one place we can see the graphs for all applications, it would b helpful.

**Comment by paularmstrong** [comment]
@abhighyaa-jain That sounds great. I don't have the time to build that kind of feature right now, so I'd love to see a pull-request for it.

**Comment by paularmstrong** [comment]
Some random implementation thoughts:

* Databases may need an extra table for `apps` and `builds` will need to reference the appropriate `app` row so they're easy to look up quickly
* Application routing will need to be updated
* Application will need a landing page to choose which app to view (maybe some other UI? maybe a default app?)
* Will probably also require #132 so that budgets can be configured more easily for each app independently

**Comment by montogeek** [comment]
@paularmstrong Would you accept a PR for this? Really interested in having it :)

**Comment by paularmstrong** [comment]
Yes, but it will definitely require more than one PR based on the above points (and unknowns)
